diff.py

```diff
@@ -2586,7 +2592,9 @@ def process(dump: str, config: Config) -> List[Line]:
         source_lines = []
 
         if config.stop_at_ret and processor.is_end_of_function(mnemonic, args):
```

Comment:
(requiring `rets_remaining` or `rets_remaining is not None` here for mypy to be happy)